pilot/pkg/model/jwks_resolver.go

```diff
@@ -107,10 +107,15 @@ type jwksResolver struct {
 	keyChangedCount uint64
 }
 
-// newJwksResolver creates new instance of jwksResolver.
-func newJwksResolver(expireDuration, evictionDuration, refreshInterval time.Duration) *jwksResolver {
+// NewJwksResolver creates new instance of jwksResolver.
+func NewJwksResolver(expireDuration, evictionDuration, refreshInterval,
```

Comment:
Maybe refactor the argument list to a struct, so that each field will have a explicit name.

Reply:
This function is not very popular (only invoked once, excluding its tests). There is also an existing struct named JWKSResolver (used inside the function to create a struct object) which has field names similar to the list of parameters, so I don't know if refactoring it by passing a struct would clarify anything. Let me know what you think :)

Reply:
Just make it easier if we want to add something else in the future.

Reply:
You do use this one in pilot, right? I'm fine with keeping argument as is, given this one is 'constructor' anyway.

Reply:
Yes. It's called twice actually. Both are in pilot.